src/utils/urls.js

```diff
@@ -38,14 +38,20 @@ function normalize (url) {
  * @param {boolean} websockets - whether the URL will be used for websockets or not
  * @returns {string} - the application base URL, containing protocol, hostname, port, and pathname
  */
-function getBaseUrl (websockets = false) {
+function getBaseUrl (websockets = false, baseonly = false) {
   const protocol = websockets
     ? window.location.protocol.startsWith('https') ? 'wss:' : 'ws:'
     : window.location.protocol
   const host = window.location.host
+  console.log('HOST', host)
```

Comment:
Removing it should fix part of the build (maybe the whole CI build?)

Reply:
Oops!